web/src/pages/loading.tsx

```diff
@@ -17,16 +17,16 @@ const LoadingPage = (): ReactElement => {
   const { updateQueue, userData } = useUserData();
   const router = useRouter();
   const { dispatch } = useContext(AuthContext);
-
   useEffect(() => {
-    if (!router.isReady) {
+    if (!router) {
```

Comment:
should this still include the previous condition? i.e.,
```js
if (!router || !router.isReady) {
     // ...
```

Reply:
isReady is not used within this component/ function. I don't think it's necessary.